src/routers/handlers/accounts_submitted/accounts.submitted.ts

```diff
@@ -0,0 +1,68 @@
+import { PrefixedUrls } from "../../../utils/constants/urls";
+import { BaseViewData, GenericHandler } from "../generic";
+import { Request, Response } from "express";
+import { getCompanyName, getCompanyNumber, getTransactionId, getUserProfile } from "./../../../utils/session";
+import { CompanyProfile } from "@companieshouse/api-sdk-node/dist/services/company-profile";
+import { checkCompanyNumberFormatIsValidate } from "./../../../utils/format/company.number.format";
+
+interface AccountsSubmittedViewData extends BaseViewData {
+        transactionId: string | Error,
+        companyProfile: Pick<CompanyProfile, "companyName" | "companyNumber">
+        payment: string
+        userEmail: string
+}
+
+export class AccountsSubmittedHandler extends GenericHandler{
+    constructor() {
+        super({
+            title: "Accounts Submitted",
+            backURL: null
+        });
+    }
+
+    async execute(req: Request, _response: Response
+    ): Promise<AccountsSubmittedViewData> {
+
+        super.populateViewData(req);
+        this.baseViewData.backURL = PrefixedUrls.PAYMENT;
```

Comment:
We don't want a back button on the submitted screen. A user should be locked out of the journey at this point.